explorer/app/components/Layout/components/Sidebar/sidebar.styles.ts

```diff
@@ -1,6 +1,7 @@
 import styled from "@emotion/styled";
 
 export const Sidebar = styled.div`
+  align-self: stretch;
```

Comment:
Required fix; I had main as a grid, and with the switch to flex this is required to stretch the sidebar to full height of `main`.

Reply:
Thank you!